webui/react/src/shared/configs/eslintrc.js

```diff
@@ -135,8 +135,10 @@ module.exports = {
         closingSlash: 'never',
       },
     ],
+    'react/jsx-uses-react': 'off',
     'react/jsx-wrap-multilines': ['error', { assignment: false, declaration: false }],
     'react/prop-types': 'off',
+    'react/react-in-jsx-scope': 'off',
```

Comment:
From https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#eslint
> If you are using [eslint-plugin-react](https://github.com/yannickcr/eslint-plugin-react), the react/jsx-uses-react and react/react-in-jsx-scope rules are no longer necessary and can be turned off or removed.